npk/src/npk.rs

```diff
@@ -375,7 +375,7 @@ impl Builder {
             context: "Failed to create temporary directory".to_string(),
             error: e,
         })?;
-        let tmp_root = copy_src_root_to_tmp(&self.root, &tmp).await?;
+        let tmp_root = copy_root_dir_to_tmp(&self.root, &tmp).await?;
```

Comment:
The copy should not be necessary if the manifest is at it's own location. Probably the whole temper is not needed anymore since the root can be packed directly from where it lives.

Reply:
The copy is now be avoided. We still need the tmp dir to write the `fs.img` squashfsh to.